### ISSUE TITLE
3D Keypoint Enhancement Training

### ISSUE THREAD

**Issue by JamesLong199** [yuangan/EAT_code#29]
Hello, thank you for your awesome research. 

I was wondering if the training code and instructions for 3D keypoint enhancement (Face-vid2vid) are included in the repo.

To my understanding, a Face-vid2vid model is first trained from scratch on the MEAD dataset following the modification specified in section 3.1.1 in the paper, before the training of A2ET. I was wondering if this understanding is correct, and if there is an ablation study on the keypoint enhancement.

Thank you in advance for your time and help!

**Comment by yuangan** [comment]
Hi, thank you for your attention. 

The training codes for 3D keypoint enhancement are not included in this repository as they are developed using a different framework. We adapted the public implementation of [OSFV(face-vid2vid)](https://github.com/zhanglonghao1992/One-Shot_Free-View_Neural_Talking_Head_Synthesis) modifying the training process as described in Section 3.1.1 of our paper.

We have detailed the ablation study of the enhanced 3D keypoints in  Section 4.6 and Table 3 of our paper, comparing it with the publicly available OSFV.

If you are interested in re-implementing the enhanced 3D keypoints, we would be pleased to offer any help we can.
